.github/workflows/nm-nightly.yml

```diff
@@ -37,7 +37,7 @@ jobs:
                             {"python":"3.9.17","label":"gcp-k8s-l4-solo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"},
                             {"python":"3.10.12","label":"gcp-k8s-l4-solo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"},
                             {"python":"3.11.4","label":"gcp-k8s-l4-solo","test":"neuralmagic/tests/test_skip_env_vars/full.txt"}]'
-            test_timeout: 480
+            test_timeout: 600
```

Comment:
nit: maybe we should just round it up to 12 hours ...

Reply:
yep updated, thanks.